src/components/Label/README.md

```diff
@@ -254,6 +254,28 @@ LANDING_BLOCK-->
 
 <!--/GITHUB_BLOCK-->
 
+## Animation
+
+<!--LANDING_BLOCK
+
+<ExampleBlock
+    code={`
+<Label animation="gradient">Content</Label>
```

Comment:
```suggestion
<Label loading>Content</Label>
```

Reply:
Sorry, forgot to fix. Fixed here https://github.com/gravity-ui/uikit/pull/2297/commits/8db583d6d261c3991c8307a053ae54e7c8a037b4